sklearn/tests/test_common.py

```diff
@@ -464,7 +464,13 @@ def test_estimators_do_not_raise_errors_in_init_or_set_params(Estimator):
 
 
 PARAM_VALIDATION_ESTIMATORS_TO_IGNORE = [
-    "DictionaryLearning",
+    "AdditiveChi2Sampler",
+    "BayesianRidge",
+    "CalibratedClassifierCV",
+    "ClassifierChain",
+    "FeatureHasher",
+    "FunctionTransformer",
+    "GenericUnivariateSelect",
     "HashingVectorizer",
```

Comment:
Currently many more entries have been reintroduced incorrectly.

Perhaps it would be easier to run `git checkout upstream/main sklearn/tests/test_common.py` and then just delete the `"DictionaryLearning"` entry.

Reply:
Looks like now there is a merge conflict _and_ there are extra edits to the list. At the very least the merge conflict must be resolved to run the CI, so please sync with the upstream main.

You can check on the [files changed](https://github.com/scikit-learn/scikit-learn/pull/24242/files) tab to see the this PR's diff with `main`. In particular, the change to the list of skippable estimators should have a diff indicating **only** the deletion of the relevant estimator(s), in this case `DictionaryLearning`. You can use that to determine when this is properly fixed.

Reply:
I made a [pull request](https://github.com/ShisuiUzumaki/scikit-learn/pull/1) in your fork with some changes which should fix things.